CMakeLists.txt

```diff
@@ -363,6 +363,14 @@ include_directories("${CMAKE_SOURCE_DIR}")
 include_directories("/usr/local/include")
 link_directories("/usr/local/lib")
 
+# Boost 1.60.0 fails on variadic templates for Apple Clang
+if(APPLE)
```

Comment:
I think modern-ish CMake installs does have the findBoost file included. I included the Apple check since unlike our Linux provision scripts we can't easily control what version of boost gets installed with `brew`. I can remove that check.